API/src/main/java/de/brandwatch/minianalytics/api/controller/QueryController.java

```diff
@@ -0,0 +1,64 @@
+package de.brandwatch.minianalytics.api.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import de.brandwatch.minianalytics.api.service.QueryService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+
+@RestController
+public class QueryController {
+
+    private static final Logger logger = LoggerFactory.getLogger(QueryController.class);
+
+    private QueryService queryService;
+
+    @Autowired
+    public QueryController(QueryService queryService) {
+        this.queryService = queryService;
+    }
+
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().create();
+
+    @PostMapping(value = "/queries")
+    public ResponseEntity query(@RequestParam String query){
+        try {
+            logger.info("POST /queries: {}", query);
+
+            return ResponseEntity.status(200).body(queryService.createQuery(query));
+        }catch (Exception e){
+            throw new ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, "", e);
+        }
+
+    }
+
+    @GetMapping(value = "/queries")
+    public ResponseEntity queries(){
+        try{
+            logger.info("GET /queries");
+
+            return ResponseEntity.status(200).body(queryService.getAllQueries());
+        }catch (Exception e){
```

Comment:
```suggestion
        } catch (Exception e) {
```

there more like this in the file. IntelliJ formatting can help you with that (opt+command+L)

Reply:
This actually only happens if I use auto formatting in IntelliJ. I don't use a whitespace after '{' but IntelliJ does. I probably have to change it somewhere.

Reply:
This is not about a whitespace after `{` though

Reply:
Then I don't see the issue

Reply:
It's missing after `}` and before `{`. You see that in the suggested change above.

Reply:
Oh... didn't know it has to be there. Doe's it always have to be there? I normally don't do it.

Reply:
Yes, that's the convention. And IntelliJ does it by default if you do autoformatting - I wonder how it doesn't for you.

Reply:
It does do it for me. Just when I write code then I don't do it. IntelliJ formatting does put a whitespace there.

Reply:
I reformatted it, so it should be fine now.